.github/workflows/pypi.yml

```
@@ -0,0 +1,29 @@
+name: Publish vembrane to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: build and publish python dists to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run:  |
+          python -m pip install --upgrade pip
+          python -m pip install poetry black flake8
+          poetry install --no-interaction
```

```suggestion
          python -m pip install poetry
```
Should suffice, right?

---

yes, indeed! copied that from the main.yml